.gitignore

```diff
@@ -1,5 +1,3 @@
 node_modules/
 package-lock.json
 haunted.js
-index.js
```

Comment:
why are these now being tracked? i think these are build artifacts, and so shouldn't be tracked into the repo

Reply:
This one was removed so it doesn't need to be ignored any more.